### PR TITLE
Remote access

### PR DESCRIPTION
**Pull request by Sigabort42**
add method remote_access
add test for method remote_access

### CODE REVIEW

File: examples/remote_access.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+"""Get the server with its id"""
```
Comment (jinxka) [inline examples/remote_access.py]: Get the server with his id

File: cbw_api_toolbox/cbw_api.py [hunk 2]
```diff
@@ -106,3 +106,12 @@ def remote_accesses(self):
         """GET request to /api/v2/remote_accesses to get all servers"""
         response = self._request("GET", [ROUTE_REMOTE_ACCESSES])
         return CBWParser().parse_response(CBWRemoteAccess, response)
+
+    def remote_access(self, remote_access_id):
+        """GET request to /api/v2/remote_accesses to get all informations about a specific server"""
+        response = self._request("GET", [ROUTE_REMOTE_ACCESSES, remote_access_id])
+        if response.status_code != 200:
```
Comment (jinxka) [inline cbw_api_toolbox/cbw_api.py]: Add newline before if

File: cbw_api_toolbox/cbw_api.py [hunk 3]
```diff
@@ -106,3 +106,12 @@ def remote_accesses(self):
         """GET request to /api/v2/remote_accesses to get all servers"""
         response = self._request("GET", [ROUTE_REMOTE_ACCESSES])
         return CBWParser().parse_response(CBWRemoteAccess, response)
+
+    def remote_access(self, remote_access_id):
+        """GET request to /api/v2/remote_accesses to get all informations about a specific server"""
```
Comment (jinxka) [inline cbw_api_toolbox/cbw_api.py]: bad comment

File: cbw_api_toolbox/cbw_api.py [hunk 4]
```diff
@@ -106,3 +106,14 @@ def remote_accesses(self):
         """GET request to /api/v2/remote_accesses to get all servers"""
         response = self._request("GET", [ROUTE_REMOTE_ACCESSES])
         return CBWParser().parse_response(CBWRemoteAccess, response)
+
+    def remote_access(self, remote_access_id):
+        """GET request to /api/v2/remote_accesses to get all informations
```
Comment (fwininger) [inline cbw_api_toolbox/cbw_api.py]: /api/v2/remote_accesses/{remote_access_id}

File: examples/remote_access.py [hunk 5]
```diff
@@ -0,0 +1,13 @@
+"""Get the server with his id"""
```
Comment (fwininger) [inline examples/remote_access.py]: remote access

File: cbw_api_toolbox/cbw_api.py [hunk 6]
```diff
@@ -106,3 +106,14 @@ def remote_accesses(self):
...
         response = self._request("GET", [ROUTE_REMOTE_ACCESSES])
         return CBWParser().parse_response(CBWRemoteAccess, response)
+
+    def remote_access(self, remote_access_id):
+        """GET request to /api/v2/remote_accesses/{remote_access_id} to get all informations
+        about a specific remote access"""
+        response = self._request("GET", [ROUTE_REMOTE_ACCESSES, remote_access_id])
+
+        if response.status_code != 200:
+            print("error remote_id")
```
Comment (fwininger) [inline cbw_api_toolbox/cbw_api.py]: remote_id does not exist !

File: examples/remote_access.py [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+
+API_KEY = ''
+SECRET_KEY = ''
+API_URL = ''
+
+CLIENT = CBWApi(API_URL, API_KEY, SECRET_KEY)
+
+REMOTE_ID = ''
+
+CLIENT.remote_access(REMOTE_ID)
```
Comment (fwininger) [inline examples/remote_access.py]: No new line

File: examples/remote_access.py [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+
+from cbw_api_toolbox.cbw_api import CBWApi
+
+API_KEY = ''
+SECRET_KEY = ''
+API_URL = ''
+
+CLIENT = CBWApi(API_URL, API_KEY, SECRET_KEY)
+
+REMOTE_ID = ''
```
Comment (fwininger) [inline examples/remote_access.py]: REMOTE_ACCESS_ID

**Pull request merged**